tests/monitoring/monitoring.go

```diff
@@ -531,42 +538,48 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", Serial, func() {
 			waitUntilAlertDoesNotExist(virtHandler.downAlert)
 		})
 
-		It("VirtApiRESTErrorsBurst should be triggered when requests to virt-api are failing", func() {
+		It("VirtApiRESTErrorsBurst and VirtApiRESTErrorsHigh should be triggered when requests to virt-api are failing", func() {
+			randVmName := rand.String(6)
```

Comment:
Previously the VM name was changed with every loop iteration but now it stays the same. Is that intentional?

Reply:
yes, just a small optimization, VM does not exist anyway, so no need to regenerate a new random name everytime